common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationAwareUTF8String.java

```diff
@@ -369,9 +371,11 @@ public static String toUpperCase(final String target, final int collationId) {
    * @return the lowercase string
    */
   public static UTF8String toLowerCase(final UTF8String target) {
```

Comment:
The same comments I made for `toUpperCase` group of methods apply to the `toLowerCase` group of methods.